patches/react-native-keychain+6.1.1.patch

```diff
@@ -96,26 +96,4000 @@ index 98e45fe..937e179 100644
  RCT_EXPORT_METHOD(resetInternetCredentialsForServer:(NSString *)server
                    resolver:(RCTPromiseResolveBlock)resolve
                    rejecter:(RCTPromiseRejectBlock)reject)
+diff --git a/node_modules/react-native-keychain/android/build/.transforms/220bd5e945f10aebb2b06c3e13aae822.bin b/node_modules/react-native-keychain/android/build/.transforms/220bd5e945f10aebb2b06c3e13aae822.bin
```

Comment:
how about this file? Why it >3000 lines?

Reply:
I'll re-apply the patch without including the build folder.